panel/chat/langchain.py

```diff
@@ -68,12 +69,13 @@ def _update_active(self, avatar: str, label: str):
             self._active_user = f"{self._active_user} - {label}"
 
     def _stream(self, message: str):
-        return self.instance.stream(
-            message,
-            user=self._active_user,
-            avatar=self._active_avatar,
-            message=self._message,
-        )
+        if message:
```

Comment:
I added the `if message` because the Pandas Agent llm streams an empty token which leads to an empty buble.